task2.py

```diff
@@ -22,22 +22,23 @@
 """
 
 
-def percent(tota, num):
-    return (100 * num) / tota
+def percent(total, num):
     # Считаем проценты
+    return (100 * num) / total
```

Comment:
Leave a comment that we believe, that total != 0

Reply:
So I tried a script where total = 0 - I added an if construction, but it doesn't work. If queries = [' '], than len(queries) (total for us here) = 1, if queries = [], than the program doesn't start. So there won't be an exepsion